lib/ModelMixins/CatalogFunctionJobMixin.ts

```diff
@@ -47,7 +47,7 @@ class FunctionJobStratum extends LoadableStratum(CatalogFunctionJobTraits) {
 
   @computed
   get shortReport() {
-    let content = "";
+    let content;
```

Comment:
If removing the default string value, can you add an explicit type definition so it is not typed as any

Reply:
Fixed.